packages/slice-machine/pages/[lib]/[sliceName]/[variation]/simulator.tsx

```diff
@@ -1,6 +1,19 @@
+import Head from "next/head";
 import Simulator from "@components/Simulator";
+import useCurrentSlice from "@src/hooks/useCurrentSlice";
 import { createComponentWithSlice } from "@src/layouts/WithSlice";
 
-const WithSlice = createComponentWithSlice(Simulator);
+const SimulatorWithSlice = createComponentWithSlice(Simulator);
 
-export default WithSlice;
+export default function SimulatorPage() {
+  const { slice } = useCurrentSlice();
+
+  return (
+    <>
+      <Head>
+        <title>Slice Simulator{slice ? ` - ${slice.model.name}` : ""}</title>
```

Comment:
I would suggest flipping this around so that it starts with slice name. This would be better when you have a lot of tabs open and the tabs are narrow. Otherwise they wil still all for example say `Slice Sim...`. This would then be `MySliceNa...`.

```suggestion
        <title>{slice ? `${slice.model.name}  - ` : ""}Slice Simulator</title>
```

Reply:
Thanks for the suggestion, @gijsbotje! We discussed this internally and came up with the following solution:

```
MySliceName - Slice Machine
Simulator: MySliceName - Slice Machine
```

The PR body includes an updated screenshot showing how this format ties in with the page titles.

Reply:
Looks great!